### ISSUE TITLE
Emoticon identified where it should not have been

### ISSUE THREAD

**Issue by CFLJacquet** [NeelShah18/emot#34]
**Describe the bug**
The emoticon `D:` is found in `"Ticket ID: 99"` where it should not have been.

**To Reproduce**
```
import emot
emot_extractor = emot.core.emot() 

text = """Ticket ID: 99"""
emoticons = emot_extractor.emoticons(text)

>>> {'value': ['D:'], 'location': [[8, 10]], 'mean': ['Sadness'], 'flag': True}
```

**Expected behavior**
Nothing should be found

**Desktop (please complete the following information):**
 - OS: ubuntu 18.04
 - Version  `emot=2.1` and `emot=3.1`

**Comment by NeelShah18** [comment]
Yep, It look like due to new pattern optimization algorithm. To create one single universal pattern makes it is really difficult find edge case. Let me see how we can make pattern which can avoid this issue. Thank you for reporting it.
